### ISSUE TITLE
PPM100

### ISSUE THREAD

**Issue by semchan** [ZHKKKe/MODNet#5]
Thanks your great work. I'm very interested in the data sets you've built.  Can you tell me how to create this dataset? Could you please open the dataset for us? thanks a lot.

**Comment by EricLe-dev** [comment]
Nope, I think they trained the model on a private dataset. There are a lot of public datasets available.

**Comment by ZHKKKe** [comment]
@semchan 
Hi! Thanks for your attention.

For your questions:
**Q1: How to create this dataset?**
We create both training set and validation set (PPM-100) by using PhotoShop.
I believe you can find many guides for obtaining alpha matte through PhotoShop on YouTube. Maybe you can refer to them.


**Q2: Could you please open the dataset for us?**
I am sorry that we can only release PPM-100 validation benchmark due to license problem.

**Comment by ntquyen11** [comment]
Hi @ZHKKKe  
I want to retrain your MODnet  on my dataset but it seems to not allow training on custom dataset, isn't it?

**Comment by ZHKKKe** [comment]
@ntquyen11 
Thanks for your attention. 
As now we just release the code of training iteration, you need to implement a data loader yourself.
